modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java

```diff
@@ -1295,6 +1297,20 @@ public String doCall(String content, File file) {
 						return content;
 					}
 
+					File modulesDir = new File(portalRootDir, "modules");
+
+					String relativePath = FileUtil.relativize(file, modulesDir);
+
+					int x = relativePath.indexOf(File.separatorChar);
```

Comment:
You can also do things a little simpler and just use `relativePath.startsWith("aspectj" + File.separator)`.